inc/Debug/DebugHelper.php

```diff
@@ -45,15 +45,15 @@ public static function dbgDeprecatedFunction($alternative = '', $callerOffset =
         if (!self::isEnabled()) return;
 
         $backtrace = debug_backtrace();
-        for ($i = 0; $i < $callerOffset; $i += 1) {
+        for ($i = 0; $i < $callerOffset; ++$i) {
             if(count($backtrace) > 1) array_shift($backtrace);
         }
 
-        list($self, $call) = $backtrace;
+        [$self, $call] = $backtrace;
 
         if (!$thing) {
             $thing = trim(
-                (!empty($self['class']) ? ($self['class'] . '::') : '') .
+                (empty($self['class']) ? ('') : $self['class'] . '::') .
```

Comment:
Good question. I think sometimes different [rules](https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md) interfere with each other? And here I think SwitchNegatedTernaryRector was applied instead of one of the Ternary Operation ones.

(we do require PHP 7.4 in master, and rector knows about that)